src/identifiers.md

```diff
@@ -4,12 +4,12 @@ r[ident]
 r[ident.syntax]
 ```grammar,lexer
 IDENTIFIER_OR_KEYWORD ->
-      XID_Start XID_Continue*
-    | `_` XID_Continue+
+      XID_START XID_CONTINUE*
+    | `_` XID_CONTINUE+
 
-XID_Start -> <`XID_Start` defined by Unicode>
+XID_START -> <`XID_Start` defined by Unicode>
 
-XID_Continue -> <`XID_Continue` defined by Unicode>
+XID_CONTINUE -> <`XID_Continue` defined by Unicode>
```

Comment:
There's a part of me that just wants to leave these two alone, given how they are defined by Unicode.  Probably I'll pull these off of the commit for now to merge the rest.